testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/CoreResourceManagementTestCase.java

```diff
@@ -974,6 +978,36 @@ public void testHostRelativeToDomainBaseDir() throws IOException {
 
     }
 
+    /** WFCORE-5960 */
+    @Test
+    public void testHostRelativeToDomainBaseDirConfiguration() throws Exception {
+        final DomainClient masterClient = domainMasterLifecycleUtil.getDomainClient();
+        final PathAddress hostAddress = PathAddress.pathAddress("host", "master");
```

Comment:
@parsharma FYI, in case it gets merged before your work in progress replacing language usage.